internal/pkg/manifest/worker_svc.go

```diff
@@ -51,70 +50,12 @@ type WorkerServiceProps struct {
 // SubscribeConfig represents the configurable options for setting up subscriptions.
 type SubscribeConfig struct {
 	Topics *[]TopicSubscription `yaml:"topics"`
-	Queue  *SQSQueue            `yaml:"queue"`
 }
 
 // TopicSubscription represents the configurable options for setting up a SNS Topic Subscription.
 type TopicSubscription struct {
-	Name    string    `yaml:"name"`
-	Service string    `yaml:"service"`
-	Queue   *SQSQueue `yaml:"queue"`
```

Comment:
Moved adding queue DS to another branch/PR to better show validation at request of @iamhopaul123